app/views/steps/caution/caution_type/edit.html.erb

```diff
@@ -1,21 +1,14 @@
 <% title t('.page_title') %>
+<% step_header %>
 
-<div class="govuk-width-container">
-  <%= step_header %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary %>
 
-  <main class="govuk-main-wrapper" id="main-content">
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_collection_radio_buttons :caution_type, @form_object.values, :value, nil %>
```

Comment:
add the following to the end: `, inline: true`
So that we keep the radio buttons as they were before, please :)